app/src/src_nonfreenet/org/breezyweather/sources/nws/NwsResultConverter.kt

```diff
@@ -182,6 +215,7 @@ private fun getDailyForecast(
 
 fun getAlerts(alerts: List<NwsAlert>?): List<Alert>? {
     if (alerts.isNullOrEmpty()) return null
+    val regex = Regex("""([0-9A-Za-z.,]) ?\n([0-9A-Za-z])""")
```

Comment:
So, this does replace only the linebreaks that were mistakenly added? And you differentiate them because they are preceeded by a space, did I understand right?

Reply:
Why not a simple `.replace(" \n", " ")`?

Reply:
What I was really checking for was a **single** `\n`, surrounded by "word characters" (i.e. `[0-9A-Za-z.,]`) on both sides.  In other words, multiple line breaks would not be replaced.

The space was optional, hence the `?`. But if I replace `" ?\n"` with `" "` then all the line breaks would disappear.